### PR TITLE
control-structures

### PR DESCRIPTION
**Pull request by kghamilton89**
* Migration of CSV functions to `control-structures.md` (document rename as well)
* Refactor of `functions.md` and `control-structures.md` to bring them up to speed with style targets.

### CODE REVIEW

File: .dictionary-other [hunk 1]
```diff
@@ -19,7 +20,9 @@ onload
...
 propertyType
 queryParameters
+roundToTen
 tagName
 unsorted
 unscale
-unscaled
\ No newline at end of file
+unscaled
+widgetConfig
```
[the remark this answers is not in the document]
Comment (rodionos) [reply .dictionary-other]: These are field/function names. Remove from dictionary, backtick at source.

```
dasharray
endlist
endfor
endscript
entityTag
getTags
ls
maxValue
metricTag
minCount
movavg
onload
postBody
propertyType
queryParameters
roundToTen
tagName
widgetConfig
```

[the remark this answers is not in the document]
Comment (kghamilton89) [reply .dictionary-other]: Removed.

[the remark this answers is not in the document]
Comment (kghamilton89) [reply .dictionary-other]: I think the reason they were added in the first place is because the lint checker was calling them errors.

**Comment by kghamilton89** [comment]
@raipc Please take a look at this Travis config. Words which are backticked or otherwise inside of backticks are still coming up as [errors](https://travis-ci.org/axibase/charts/builds/416250598#L529). Adding them to the dictionary is undesirable, since they should be passed over due to backticks.

**Comment by raipc** [comment]
@kghamilton89 This is a speller error. It seems like it recognizes code blocks only if the triple backticks are in the start of the line. If the backticks are prepended by space, the content is not ignored.

I think we need a rule to validate that validates such cases.

**Comment by kghamilton89** [comment]
@raipc This is what I believe as well. It only occurs when we have something like:

`some text

some text`

**Comment by raipc** [comment]
There should be no space before and after backticks. The speller worked wrong because of the following pattern:
  ``` json

**Comment by kghamilton89** [comment]
@raipc Good find. I searched for ``` ls because I thought that may have been the cause but I didn't think to check json tag as well

**Pull request merged**